pkg/skaffold/config/options.go

```diff
@@ -91,6 +90,7 @@ type SkaffoldOptions struct {
 	RPCPort            int
 	RPCHTTPPort        int
 	BuildConcurrency   int
+	MakePathsAbsolute  *bool
```

Comment:
Why make it this a *bool? Would `opts. MakePathsAbsolute` false not suffice?

This is not a command line flag, right?

Reply:
> Why make it this a *bool? Would opts. MakePathsAbsolute false not suffice?
I have 3 modes of parsing (see description). So nillable boolean `nil, true, and false` works out. Otherwise I can use an explicit struct for `parsingMode`.

Reply:
So, true is set in diagnose workflow. Its nil by default.
When is `opts. MakePathsAbsolute` set to false?  - Do you plan to set it to false in inspect workflow?

Reply:
yes, it'll be set to false for `skaffold inspect` commands